Source/Csla/ApplicationContext.cs

```diff
@@ -187,7 +187,7 @@ public enum ExecutionLocations
     /// for all explicit object serialization (such as cloning,
     /// n-level undo, etc).
     /// </summary>
-    public static Type SerializationFormatter { get; internal set; } = typeof(Serialization.Mobile.MobileFormatter);
+    public static Type SerializationFormatter { get; internal set; } = null;
```

Comment:
Is there a reason why this is not put properly into the DI now with the rework?

Reply:
I'll look. The formatter factory is, iirc, a static, so the question will be whether every place in the framework that uses the factory will have access to DI.

Reply:
Yes, doable. Then the only static property left is one that can probably be removed, as it is to allow fallback from expression trees to reflection for method calling, and that capability hasn't been necessary since Blazor first came out without expression trees.

Reply:
Sounds awesome. Should make some tests also easier due to less shared statics :)

Reply:
Here's a very simple custom serializer so you can see they work:

https://github.com/MarimerLLC/csla/blob/88d55a96b039df15f9867ab6542c0ee446df5609/Source/Csla.test/Serialization/CustomSerializerTests.cs#L109

You do have me rethinking how a type is mapped to a serializer though. Right now I'm doing it with a direct type comparison, not counting inheritance or interfaces. I suppose I could broaden it so a serializer gets registered for types that can be derived from the type in the map?

Right now:

* MyType => MyTypeSerializer

But maybe:

* IMyType => MyTypeSerializer
* MyTYpe : IMyType=> MyTypeSerializer
* SomeotherType : IMyType => MyTypeSerializer

Reply:
There should be no shared statics anymore, unless there's still a test for that old reflection code - but even if that exists it would be highly specialized and isolated.